src/Protocol/OStatus.php

```diff
@@ -1256,10 +1257,16 @@ private static function addHeader($doc, $owner)
 		$root->setAttribute("xmlns:statusnet", NAMESPACE_STATUSNET);
 		$root->setAttribute("xmlns:mastodon", NAMESPACE_MASTODON);
 
-		$attributes = array("uri" => "https://friendi.ca", "version" => FRIENDICA_VERSION."-".DB_UPDATE_VERSION);
+		switch ($filter) {
+			case 'activity': $title = t('%s\'s timeline', $owner['name']); break;
```

Comment:
Is this single liner PSR-2 compliant?

Reply:
It doesn't say anything that precise about switched.

Reply:
And it looks ugly :-)